<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CadastroProduto</class>
 <widget class="QDialog" name="CadastroProduto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastro de Produto</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="logoHorizontalLayout">
     <item>
      <spacer name="leftSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="logoLabel">
       <property name="minimumSize">
        <size>
         <width>264</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>264</width>
         <height>100</height>
        </size>
       </property>
       <property name="pixmap">
        <pixmap resource="resources.qrc">:/imagens/logo.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="rightSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="tituloLabel">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>85</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="PlaceholderText">
         <brush brushstyle="SolidPattern">
          <color alpha="128">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>85</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="PlaceholderText">
         <brush brushstyle="SolidPattern">
          <color alpha="128">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="PlaceholderText">
         <brush brushstyle="SolidPattern">
          <color alpha="128">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="styleSheet">
      <string>
       QLabel { font-size: 20px; font-weight: bold; color: black; }
      </string>
     </property>
     <property name="text">
      <string>Cadastro de Produto</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="nomeLineEdit">
     <property name="placeholderText">
      <string>Nome do produto</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="grupoComboBox">
     <item>
      <property name="text">
       <string>Selecione o grupo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cabelos</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pele</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Maquiagem</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Unhas</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Corpo</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="codigoBarrasLineEdit">
     <property name="placeholderText">
      <string>Código de barras</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="marcaLineEdit">
     <property name="placeholderText">
      <string>Marca</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="descricaoTextEdit">
     <property name="placeholderText">
      <string>Descrição do produto</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="quantidadeSpinBox">
     <property name="prefix">
      <string>Qtd: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="precoDoubleSpinBox">
     <property name="prefix">
      <string>R$ </string>
     </property>
     <property name="minimum">
      <double>-2.000000000000000</double>
     </property>
     <property name="maximum">
      <double>99999.990000000005239</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cadastrarButton">
     <property name="styleSheet">
      <string>
       QPushButton {
         background-color: #FF66B2;
         color: white;
         border-radius: 15px;
         border: 2px solid #FF3399;
         padding: 10px 20px;
         font-size: 16px;
       }
       QPushButton:hover {
         background-color: #FF3399;
         border-color: #FF66B2;
       }
      </string>
     </property>
     <property name="text">
      <string>Cadastrar Produto</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="messageLabel">
     <property name="styleSheet">
      <string>
       QLabel {
         color: red;
         font-size: 16px;
         font-weight: bold;
       }
      </string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
